src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  position: relative;
+  max-width: 1200px;
```

Comment:
do not set it, let's it be full-width
moreover this container is redundant - add styles you need to `.header`